src/telemetry/userTracker.ts

```diff
@@ -0,0 +1,109 @@
+import { formatISO, startOfWeek } from 'date-fns'
```

Comment:
This file duplicates logic from the sourcegraph/sourcegraph repo: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@90826c6d7370511469e528258ee6ee446f784870/-/blob/client/shared/src/telemetry/web/userTracker.ts